Fix wrong type for function id (replace double by size_t).

--- ./include/roboptim/core/finite-difference-gradient.hh~ 2011-10-10 11:56:24.000000000 +0200
+++ ./include/roboptim/core/finite-difference-gradient.hh  2010-12-07 16:48:32.000000000 +0100
@@ -103,7 +103,7 @@
        Function::value_type epsilon,
        Function::result_t& gradient,
        const Function::argument_t& argument,
-       Function::value_type idFunction) const throw ();
+       Function::size_type idFunction) const throw ();
     };
 
     /// \brief Precise finite difference gradient computation.
@@ -117,7 +117,7 @@
        Function::value_type epsilon,
        Function::result_t& gradient,
        const Function::argument_t& argument,
-       Function::value_type idFunction) const throw ();
+       Function::size_type idFunction) const throw ();
     };
   } // end of namespace policy.
 
